client/src/views/About.vue

```diff
@@ -128,12 +151,46 @@ export default {
     drawCard() {
       unoService.drawCard(this.game_id, this.username)
         .then(this.updateData());
-    }
+    },
+    orgOff() {
+      if (this.sortByColor == true || this.sortByNum == true) {
+        this.loadingHand = true;
+      }
+      this.sortByNum = false;
+      this.sortByColor = false;
+    },
+    orgByNum() {
+      this.sortByNum = true;
+      this.sortByColor = false;
+
+      var colors = { 'red': 0, 'blue': 1 , 'green': 2, 'yellow': 3 }
+      this.cards.sort((a, b) => { return colors[a.color] - colors[b.color];});
+
+      return this.cards.sort((a, b) => { return a.number - b.number;});
```

Comment:
Shouldn't this be `return a.number < b.number` both here and in the other sorts?

Reply:
That is a valid argument, haha! Idk, I just follow StackOverFlow sometimes and if it works perfectly then I don't fight it. haha